app/forms/provider_interface/reasons_for_rejection_wizard.rb

```diff
@@ -2,27 +2,31 @@ module ProviderInterface
   class ReasonsForRejectionWizard
     include ActiveModel::Model
 
+    TRANSLATION_KEY_PREFIX = 'activemodel.errors.models.provider_interface/reasons_for_rejection_wizard.attributes'.freeze
+
     class NestedAnswerValidator < ActiveModel::EachValidator
       def validate_each(record, attribute, value)
         return unless options.key?(:collection_name) && options.key?(:selected_option)
 
         top_level_question = options[:top_level_question]
         top_level_answer = record.send(top_level_question)
 
-        record.errors.add(top_level_question, :blank) if top_level_answer.blank?
-        record.errors.add(top_level_question, :inclusion) unless %w[Yes No].include?(top_level_answer)
```

Comment:
might have accidentally removed this error?

Reply:
It was intentional though debatable as to whether it's the right thing to do.
The element which sends these values is always a radio button so validating blank is enough in practical terms for normal use. 
If someone were to craft a request with a non Yes/No param value then this validation could be used, IMO it was unnecessary but open to opinion on this.

Reply:
yea that is a good point, we'll prob have to do this everywhere else otherwise 👍